ClassFieldTheory/Cohomology/FiniteCyclic/CubeLemma.lean

```diff
@@ -0,0 +1,31 @@
+import Mathlib.Combinatorics.Quiver.ReflQuiver
```

Comment:
This file is in the wrong place -- can you move it to some ClassFieldTheory/Mathlib directory?